isis/CMakeLists.txt

```diff
@@ -33,15 +33,15 @@ set(PACKAGE            "ISIS")
 set(PACKAGE_NAME       "USGS ISIS")
 
 # Version number
-set(VERSION            "3.9.0")
+set(VERSION            "3.9.1")
 set(PACKAGE_VERSION    ${VERSION})
 
 # Full name and version number
 set(PACKAGE_STRING     "${PACKAGE_NAME} ${VERSION}")
 
 # Other release information
-set(VERSION_DATE              "2019-09-13")
-set(RELEASE_STAGE             "beta") # (alpha, beta, stable)
+string(TIMESTAMP VERSION_DATE "%Y-%m-%d" UTC)
```

Comment:
Generates the same string as before, just uses the current date when cmake is configured.